CONTRIBUTING.md

```diff
@@ -0,0 +1,249 @@
+
+Updated outdated dependencies to their latest version. For more information, view [PNPM documentation](https://pnpm.io/cli/update).
+
+```bash
+pnpm update-outdated-deps
+```
+
+## CI
+
+We use [GitHub Actions](https://docs.github.com/en/actions) for this repository.
+
+You can find the configuration is in the `.github/workflows` folder and the build results are available [here](https://github.com/workleap/wl-web-configs/actions).
+
+We currently have 2 builds configured:
+
+### Changesets
+
+This action runs on a push on the `main` branch. If there is a file present in the `.changeset` folder, it will publish the new package version on npm.
+
+### CI
+
+This action will trigger when a commit is done in a PR to `main` or after a push to `main`. The action will run `build`, `lint` and `test` commands on the source code.
+
+## Add a new package to the monorepo
+
+There are a few steps to add new packages to the monorepo.
+
+Before you add a new package, please read the [GSoft GitHub guidelines](https://github.com/workleap/github-guidelines#npm-package-name).
+
+### Create the package
+
+First, create a new folder matching the package name in the [packages](/packages) directory.
+
+Open a terminal, navigate to the newly created directory, and execute the following command:
+
+```bash
+pnpm init
+```
+
+Answer the CLI questions.
+
+Once the *package.json* is generated, please read again the [GSoft GitHub guidelines](https://github.com/workleap/github-guidelines#npm-package-name) and make sure the package name, author and license are valid.
+
+Don't forget to add the [npm scope](https://docs.npmjs.com/about-scopes) *"@workleap"* before the package name. For example, if the project name is "foo", your package name should be "@workleap/foo".
+
+Make sure the package publish access is *public* by adding the following to the *package.json* file:
+
+```json
+{
+  "publishConfig": {
+    "access": "public"
+  }
+}
+```
+
+### Dependencies
+
+npm *dependencies* and *peerDependencies* must be added to the package own *package.json* file.
+
+## Maintainers notes
+
+### TS moduleResolution
+
+Packages compiled for CommonJS (`browserlists-config`, `eslint-plugin`, `stylelint-configs`) cannot use `moduleResolution: "NodeNext"` because with TS 5.2, it requires to be used in conjunction with `module: "NodeNext"` (but we want `module: "CommonJS"`).
+
+Those packages also cannot use `moduleResolution: "Bundler"` because it requires to use at a minimum `module: "ES2015"`, but again, we want `module: "CommonJS"`.
+
+Therefore, we must use `moduleResolution: "Node"`, which is the equivalent of `Node v10`. A `Node v10` environment doesn't support an `"exports"` field in the `package.json` file according to this [issue](https://github.com/microsoft/TypeScript/issues/51862#issuecomment-1358049778).
+
+Consequently, we must add the `"types"` field to the `package.json` file of the projects that are also compiled for CommonJS.
+
+ESM only package:
```

Comment:
I don't think those are necessary for this package? Added it to the web-configs repo because it was dealing with CommonsJS and ESM but I don't see a cause for this repo